mne/decoding/time_gen.py

```diff
@@ -764,7 +849,8 @@ class GeneralizationAcrossTime(_GeneralizationAcrossTime):
         If an integer is passed, it is the number of folds.
         Specific cross-validation objects can be passed, see
         scikit-learn.cross_validation module for the list of possible objects.
-        Defaults to 5.
+        If clf is a classifier, defaults to KFold(n_folds=5), else defaults to
+        StratifiedKFold(n_folds=5).
```

Comment:
Ah ... My point is simply that right now, StratifiedKFold and KFold are in the wrong order in the doc.